Sources/Amplitude/Plugins/iOS/UIKitElementInteractions.swift

```diff
@@ -93,6 +94,13 @@ extension UIApplication {
     @objc func amp_sendAction(_ action: Selector, to target: Any?, from sender: Any?, for event: UIEvent?) -> Bool {
         let sendActionResult = amp_sendAction(action, to: target, from: sender, for: event)
 
+        // TODO: Reduce SwiftUI noise by finding the unique view that the action method is attached to.
+        // Currently, the action methods pointing to a SwiftUI target are blocked.
+        let targetClass = NSStringFromClass(type(of: target as AnyObject))
```

Comment:
from some of the session replay work `String(cString: object_getClassName(target))` is significantly faster

Reply:
may still need some AnyObject conversion

Reply:
This works without any AnyObject conversion!